source/Eyetracking.NET/Eyetracker.cs

```diff
@@ -1,18 +1,56 @@
-﻿using Eyetracking.NET.Tobii;
+﻿using System;
+using Eyetracking.NET.Tobii;
 
 namespace Eyetracking.NET
 {
-    public class Eyetracker 
+    public class Eyetracker : IEyetrackerVr, IEyetracker
```

Comment:
If I write an application using this class, how can I know if it  is using a VR tracker or a desktop tracker? I guess it opens up for a bit of unexpected behavior.

Reply:
Yes, it might be better to always have them separated.